Makefile

```diff
@@ -186,7 +186,7 @@ chat2: | build deps librln
 
 rln_keystore_generator: | build deps librln
```

Comment:
I'd go for `rln-keystore-generator`. General Makefile convention is that words in target names are not separated at all or separated by a hyphen. See https://style-guides.readthedocs.io/en/latest/makefile.html#targets